<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1175</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 11pt &quot;华光行书_CNKI&quot;;</string>
   </property>
   <property name="text">
    <string>Video monitoring equipment interface</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 11pt &quot;华光行书_CNKI&quot;;</string>
   </property>
   <property name="text">
    <string>Time slots</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 11pt &quot;华光行书_CNKI&quot;;</string>
   </property>
   <property name="maximumDate">
    <date>
     <year>2021</year>
     <month>12</month>
     <day>31</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>yyyy/M/d HH</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 11pt &quot;华光行书_CNKI&quot;;</string>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>50</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 11pt &quot;华光行书_CNKI&quot;;</string>
   </property>
   <property name="text">
    <string>Device status:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>130</y>
     <width>281</width>
     <height>421</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 11pt &quot;华光行书_CNKI&quot;;</string>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 11pt &quot;华光行书_CNKI&quot;;</string>
    </property>
    <property name="text">
     <string>Installation time</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 11pt &quot;华光行书_CNKI&quot;;</string>
    </property>
    <property name="text">
     <string>Latitude and longitude</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_lo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 11pt &quot;华光行书_CNKI&quot;;</string>
    </property>
    <property name="text">
     <string>E:123.456</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_la">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 11pt &quot;华光行书_CNKI&quot;;</string>
    </property>
    <property name="text">
     <string>W:103.794</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_time">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 11pt &quot;华光行书_CNKI&quot;;</string>
    </property>
    <property name="text">
     <string>2021/01/01</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ID">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_inspection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 11pt &quot;华光行书_CNKI&quot;;</string>
    </property>
    <property name="text">
     <string>Number of the inspection point:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_finish">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>380</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 11pt &quot;华光行书_CNKI&quot;;</string>
    </property>
    <property name="text">
     <string>change finish</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>109</y>
     <width>771</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>40</y>
     <width>87</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 11pt &quot;华光行书_CNKI&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>on</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>off</string>
    </property>
   </item>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>771</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>460</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 11pt &quot;华光行书_CNKI&quot;;</string>
   </property>
   <property name="text">
    <string>Play/Pause</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_e">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>450</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 11pt &quot;华光行书_CNKI&quot;;</string>
   </property>
   <property name="text">
    <string>00:00:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_b">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>450</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 11pt &quot;华光行书_CNKI&quot;;</string>
   </property>
   <property name="text">
    <string>00:00:00</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
